README.md

```diff
@@ -105,29 +106,49 @@ https://wireless.murata.com/type-abz-078.html
 
 - LORA is not enabled by default. You need to update your local mbed_app.json file:
 
+```json
+{
+    "target_overrides": {
+        "MTB_MURATA_ABZ": {
+            "target.components_add": ["SX1276"]
+        }
+    }
+}
 ```
+
+If you need to Low Power with VDD TXCO like [b-l072z-lrwan1](https://www.st.com/en/evaluation-tools/b-l072z-lrwan1.html), then you need JP9 positionned 1-2.
+And reflect the pin used for that because it's defined to `NC` by default.
+
+```json
 {
     "target_overrides": {
         "MTB_MURATA_ABZ": {
-            "target.components_add":            ["SX1276"],
-            "sx1276-lora-driver.spi-mosi":       "PA_7",
-            "sx1276-lora-driver.spi-miso":       "PA_6",
-            "sx1276-lora-driver.spi-sclk":       "PB_3",
-            "sx1276-lora-driver.spi-cs":         "PA_15",
-            "sx1276-lora-driver.reset":          "PC_0",
-            "sx1276-lora-driver.dio0":           "PB_4",
-            "sx1276-lora-driver.dio1":           "PB_1",
-            "sx1276-lora-driver.dio2":           "PB_0",
-            "sx1276-lora-driver.dio3":           "PC_13",
-            "sx1276-lora-driver.txctl":          "PC_2",
-            "sx1276-lora-driver.rxctl":          "PA_1",
-            "sx1276-lora-driver.pwr-amp-ctl":    "PC_1",
-            "sx1276-lora-driver.tcxo":           "PA_12"
```

Comment:
Was it expected to remove this part ?
How SX1276 is now configured ?

Reply:
Pushed to fast sorry (making 10 things in the same time), Just need to change names with the one here in [PinNames.h](https://github.com/ARMmbed/stm32customtargets/blob/master/TARGET_STM32L0/TARGET_MTB_MURATA_ABZ/PinNames.h#L146-#L157), I'll ping when it's done

Reply:
Ok fixed